### DISCUSSION TITLE
Why do the CoreUI success elements use dark text?

### DISCUSSION THREAD

**Discussion by dataclear** [coreui/coreui#816] [General]
I've noticed the `btn-success` (and most other `*-success` elements) now use a dark text, this is different to Bootstrap where `success` text is white. Is there a reason for this?

I know the contrast is slightly higher for black text bit in my opinion it detracts from the use of the button, looking out of place as a positive element and makes the user experience worse.

Has this change been made on purpose?

**Comment by mrholek** [comment]
@dataclear We use different color sets than Bootstrap, and to comply with accessibility rules, we have dark text in this case. We prefer white text, but the contrast ratio must be at least 4.5:1.
